acceptance/testdata/analyzer/analyze-image/Dockerfile

```diff
@@ -19,3 +19,11 @@ RUN chown -R $CNB_USER_ID:$CNB_GROUP_ID /layers
 
 # ensure docker config directory is root owned and NOT world readable
 RUN chown -R root /docker-config; chmod -R 700 /docker-config
+
+ARG registry
```

Comment:
Do we need to update the corresponding Dockerfile.windows now?

Reply:
Probably. I need to figure out how to re-write it for windows, but I uncommented all the tests that were skipping windows so I can see them fail and fix it :)

Reply:
Alright - I finally fixed up windows acceptance tests.